tests/wpt/tests/css/css-overflow/scrollable-overflow-transform-005.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Overflow: Scrollable Overflow Transform Exclusion of Untransformed Child Box</title>
+<link rel="author" title="Jo Steven Novaryo" href="mailto:steven.novaryo@gmail.com">
+<link rel="help" href="https://drafts.csswg.org/css-overflow-3/#scrollable" />
+<meta name="assert" content="Checks that the untransformed child box is not included in scrollable overflow.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/check-layout-th.js"></script>
+<style>
+  .container {
+    width: 100px;
+    height: 100px;
+    overflow: scroll;
+    background: silver;
+    border: solid thick;
+  }
+
+  .element {
+    width: 200px;
+    height: 200px;
+    background: lime;
+  }
+
+  .element-2 {
+    width: 100px;
+    height: 100px;
+  }
+</style>
+<body onload="checkLayout('.container')">
+
+  <div class="container" data-expected-scroll-width="100" data-expected-scroll-height="100">
+    <div style="transform: translate(-200px, -200px);" class="element">
+    </div>
+  </div>
```

Comment:
https://drafts.csswg.org/cssom-view/#scrolling-area isn't clear about taking into account the transforms of the contents. Probably it should refer to https://drafts.csswg.org/css-overflow-3/#scrollable?

I think you should file an issue to the CSSWG, an link to it in the test with `<link rel="help"`

WebKit has the desired behavior, but is still failing the test when using classical scrollbars. If the scrollbar is 15px wide, then `scrollWidth` will be `85px`. I guess you can address this with `scrollbar-width: none`.

Reply:
I am not sure about what is [scrolling-area](https://drafts.csswg.org/cssom-view/#scrolling-area) in this context, I guess the `<link rel="help"` already referring to https://drafts.csswg.org/css-overflow-3/#scrollable?

Noted, I will file an issue soon. Marked the ones with unusual behavior as tentative.

Fixed scrollbar width, thank you for the review!